### PR TITLE
new craftable sunglasses

### PR DESCRIPTION
**Pull request by Tlaltecuhtli**
:cl: 
add: diagnostic sunglasses
add: nightvision sunglasses(not craftable)
fix: the medhudsunglasses color were brighter than the inhand object, now the sprite is less bright when worn
/:cl:
now we have all the hud glasses as sunglasses

test if it works ingame: to be done

### CODE REVIEW

**Comment by Mark9013100** [comment]
diagnostic sunglasses yes
nightvision sunglasses no, NV or flash protection, pick one

File: code/modules/clothing/glasses/_glasses.dm [hunk 1]
```diff
@@ -116,6 +116,13 @@
 	user.visible_message("<span class='suicide'>[user] is tightening \the [src]'s straps around [user.p_their()] neck! It looks like [user.p_theyre()] trying to commit suicide!</span>")
 	return OXYLOSS
 
+/obj/item/clothing/glasses/night/sunglasses
+	name = "night vision sunglasses
+	des = "Protects from moonlight."
```
Comment (81Denton) [inline code/modules/clothing/glasses/_glasses.dm]: I mean technically he's not wrong

**Comment by ExcessiveUseOfCobblestone** [comment]
nvgs are icky but i like the other hudglasses

**Comment by davethwave** [comment]
I would upvote this normally but this will make revs nearly unplayable.

**Comment by BeeSting12** [comment]
How the fuck are you supposed to see in the dark and not get blinded by a flash. 

:-1:

NVGs should give bonus blind time not protection.

**Comment by Tlaltecuhtli** [comment]
night vision sunglasess are no longer available to mortal beings

**Comment by tralezab** [comment]
can someone explain to me how night vision sunglasses work

**Comment by octareenroon91** [comment]
https://en.wikipedia.org/wiki/High-dynamic-range_imaging

**Comment by Kagus** [comment]
> can someone explain to me how night vision sunglasses work

It's like, uhh... It's like those noise-cancelling headphones that hunters wear, yeah? Amplifies low-volume sounds and then automatically detects and cancels out high-volume spikes! Yeah, that's it!

**Comment by zxaber** [comment]
>can someone explain to me how night vision sunglasses work

Take the lens from an  [auto-darkening welding hood](https://en.wikipedia.org/wiki/Welding_helmet#Auto-darkening_filters), cut it to the same shape as the NVG eye holes, and glue it on.

Memes aside, the diagnostics sunglasses sound cool, though I don't know that I'm sold on further removing flashes as viable stun weapons.

**Comment by zxaber** [comment]
I'm super biased, because I main Robo with a fallback on borg.

But for most borgs, the only real tool available to non-harmfully de-escalate a fight is their flash. I'm certainly not against protection from flashes, as it does make some fights more interesting when you can't just stun everyone and drag the fighters apart. But I worry about it becomming too commonplace and putting borg players in a bind *too* often.

**Comment by optimumtact** [comment]
random unasked for buff with no real reason.

**Comment by Tlaltecuhtli** [comment]
@optimumtact this doesnt make you create new sunglasses it just make you able to put diagnostic hud on already existing sunglasses, it already exist for sec and medical not sure how diag hud is a buff

**Comment by optimumtact** [comment]
remove this item completely

**Pull request closed without merging**